src/pudl/metadata/resources/ferc1.py

```diff
@@ -413,6 +413,28 @@
         "etl_group": "glue",
         "field_namespace": "ferc1",
     },
+    "utilities_ferc1_dbf": {
+        "description": "This table maps the assign utility ID FERC1 to the native utility ID from the FERC1 DBF inputs - originally reported as respondent_id.",
+        "schema": {
+            "fields": ["utility_id_ferc1", "utility_id_ferc1_dbf"],
+            "primary_key": ["utility_id_ferc1"],
+            "foreign_key_rules": {"fields": [["utility_id_ferc1_dbf"]]},
```

Comment:
hm i think you are right on the PK. because these ferc util "source" (xbrl or dbf) id's have a one:one relationship with the `utility_id_ferc1`, this wasn't failing. but i think the correct thing to do here is have the primary key be the `utility_id_ferc1_{data_source}`

Reply:
I swapped the PK and FK for these two tables!